tests/networks/nets/test_hyena_nd_unetr.py

```
@@ -0,0 +1,165 @@
+                use_hyena=True,
+            )
+
+    @skipUnless(HAS_NVSUBQ, "Requires nvsubquadratic")
+    def test_duplicate_hyena_stages_kwarg_rejected(self):
+        with self.assertRaisesRegex(TypeError, "hyena_stages"):
+            HyenaNDUNETR(
+                in_channels=1,
+                out_channels=14,
+                feature_size=12,
+                hyena_stages=(True, True, False, False),
+                **{"hyena_stages": (True, False, True, False)},  # type: ignore[arg-type]
```

Ruff (0.15.18)

[error] Repeated keyword argument: `hyena_stages`

(PLE1132)